rep-0105.rst

```diff
@@ -51,38 +79,61 @@ therefore eliminating drift, but causing discrete jumps when new
 sensor information arrives.
 
 The ``map`` frame is useful as a long-term global reference, but
-discrete jumps make it a poor reference frame for local sensing and
+discrete jumps in position estimators make it a poor reference frame for local sensing and
 acting.
 
+**Map Conventions**
 
-odom
-''''
+Map coordinate frames can either be referenced globally or to an application specific position.
+A example of an application specific positioning might be Mean Sea Level [#MSL]_ according to EGM1996 [#EGM96]_ such that the z position in the map frame is equivalent to meters above sea level.
+Whatever the choice is the most important part is that the choice of reference position is clearly documented for users to avoid confusion.
 
-The coordinate frame called ``odom`` is a world-fixed frame. The pose
-of a mobile platform in the ``odom`` frame can drift over time,
-without any bounds. This drift makes the ``odom`` frame useless as a
-long-term global reference. However, the pose of a robot in the
-``odom`` frame is guaranteed to be continuous, meaning that the pose
-of a mobile platform in the ``odom`` frame always evolves in a smooth
-way, without discrete jumps.
+When defining coordinate frames with respect to a global reference like the earth:
+ * The default should be to align the x-axis east, y-axis north, and the z-axis up at the origin of the coordinate frame.
+ * If there is no other reference the default position of the z-axis should be zero at the height of the WGS84 ellipsoid.
 
-In a typical setup the ``odom`` frame is computed based on an odometry
-source, such as wheel odometry, visual odometry or an inertia
-measurement unit.
+In the case that there are application specific requirements for which the above cannot be satistfied as many as possible should still be met.
 
-The ``odom`` frame is useful as an accurate, short-term local
-reference, but drift makes it a poor frame for long-term reference.
+An example of an application which cannot meet the above requirements is a robot starting up without an external reference device such as a GPS, compass, nor altimeter.
+But if the robot still has an accelerometer it can intialize the map at it's current location with the z axis upward.
```

Comment:
Utterly pedantic, so feel free to ignore: no apostrophe in the possessive form of "its".